### ISSUE TITLE
Windows install docs, bzip2 not available

### ISSUE THREAD

**Issue by MartinJepsen** [mamba-org/mamba#2940]
### Troubleshooting docs

- [X] My problem is not solved in the Troubleshooting docs

### How did you install Mamba?

Other (please describe)

### Search tried in issue tracker

bzip2, windows bzip2

### Latest version of Mamba

- [X] My problem is not solved with the latest version

### Tried in Conda?

Not applicable

### Describe your issue

The docs for Windows installation ([link](https://mamba.readthedocs.io/en/latest/micromamba-installation.html#windows)) recommend PowerShell for install, but Windows does not have native support for bzip2, so the command 

```powershell
tar xf micromamba.tar.bz2
```

will not work. It outputs

```txt
tar.exe: Error opening archive: Can't initialize filter; unable to run program "bzip2 -d"
```

**Comment by jonashaag** [comment]
Is this not an option for you? https://mamba.readthedocs.io/en/latest/micromamba-installation.html#install-script

**Comment by jonashaag** [comment]
I think we could change those URLs to uncompressed ones though

**Comment by MartinJepsen** [comment]
> Is this not an option for you? https://mamba.readthedocs.io/en/latest/micromamba-installation.html#install-script

Yes it is, but the issue is about the manual install documentation recommending PowerShell which won't work.

**Comment by MartinJepsen** [comment]
> I think we could change those URLs to uncompressed ones though

I think that would be a good option, because then Windows users can install micromamba with PowerShell only, meaning that they won't need to install either Git Bash or something that can handle bzip2.

**Comment by jonashaag** [comment]
If I'm not mistaken you can try the URLs from the install.sh script

**Comment by nmishra93** [comment]
@jonashaag
I would like to add the updated installation instructions for Windows PowerShell such that there is no need for Git Bash as PowerShell by default supports bzip2. The updated code installs micromamba using PowerShell. It will fix the issue raised by @MartinJepsen.

# Download the binary
`Invoke-Webrequest -URI https://micro.mamba.pm/api/micromamba/win-64/latest -OutFile micromamba.tar.bz2`
`mkdir "$HOME\micromamba"`
`bzip2 -d micromamba.tar.bz2`
`tar xf micromamba.tar -C "$HOME\micromamba"`

# Remove the tar archive
`Remove-Item micromamba.tar`

# Invoke the hook
`& "$HOME\micromamba\Library\bin\micromamba.exe" shell hook -s powershell | Out-String | Invoke-Expression`

# and use micromamba directly
`micromamba create -f ./test/env_win.yaml -y`
`micromamba activate yourenv`

**Comment by jonashaag** [comment]
Please send a PR, thank you

**Comment by jonashaag** [comment]
But we don't need tar/bzip2 if we use the uncompressed URL (see install.sh script`0

**Comment by MartinJepsen** [comment]
I actually ended up using the instructions provided in the README for micromamba-releases: https://github.com/mamba-org/micromamba-releases

Works perfect

**Comment by aleskunder** [comment]
Please fix this, I encountered this just now when trying to show someone how awesome micromamba is and it's such a shame to encounter

**Comment by floriandeboissieu** [comment]
I tried with uncompressed link, e.g. https://github.com/mamba-org/micromamba-releases/releases/latest/download/micromamba-win-64, without any success:
```
Invoke-Webrequest -URI https://github.com/mamba-org/micromamba-releases/releases/latest/download/micromamba-win-64 -OutFile micromamba.exe
.\micromamba.exe --help # gives no answer
```
